backend/routes/auth.js

```diff
@@ -173,9 +173,17 @@ module.exports = function (app) {
 
   app.post('/auth/logout', logoutHandler)
 
-  // TODO: Should this at least use API key auth?
   app.post('/auth/registration', async (req, res) => {
-    const { seller, status, error } = await createSeller(req.body)
+    if ((await numSellers()) > 0) {
+      return res.status(status).json({
```

Comment:
status is not set at that point. and anyway it should just return a 200 which I think it does by default so perhaps just change this to 
`return res.json({...})`

Reply:
Done. Changed to a return a 409 to be consistent with other dshop code's behavior when trying to create a duplicate.